EventFilter/CTPPSRawToDigi/src/RawDataUnpacker.cc

```diff
@@ -80,11 +80,13 @@ int RawDataUnpacker::ProcessOptoRxFrame(const word *buf, unsigned int frameSize,
   #endif
 
   // parallel or serial transmission?
-  if (FOV == 1)
+  if (FOV == 1) {
     return ProcessOptoRxFrameSerial(buf, frameSize, fc);
+  }
 
-  if (FOV == 2)
+  if (FOV == 2 || FOV == 3) {
     return ProcessOptoRxFrameParallel(buf, frameSize, fedInfo, fc);
+  }
 
   if (verbosity)
     LogWarning("Totem") << "Error in RawDataUnpacker::ProcessOptoRxFrame > " << "Unknown FOV = " << FOV << endl;
```

Comment:
it looks like the independent ifs starting from ```if (FOV == 1) ``` are better handled with a switch-case or with an if-else